content/en/docs/tasks/inject-data-application/podpreset.md

```diff
@@ -26,157 +23,289 @@ Get an overview of PodPresets at
 {{% capture steps %}}
 
 
-## Simple Pod Spec Example
+## Use Pod presets to inject environment variables and volumes
 
-This is a simple example to show how a Pod spec is modified by the Pod
-Preset.
+In this exercise, you create a Pod preset that has one environment variable and a volume mount.
+Here is the configuration file for the PodPreset:
 
 {{< codenew file="podpreset/preset.yaml" >}}
 
-Create the PodPreset:
+In the configuration file, you can see that the preset has an environment variable definition called `DB_PORT`
+and a volume mount definition called `cache-volume` which is mounted under `/cache`. The `selector` specifies that
+the preset will act upon any Pod that is labeled `role:frontend`.
+
+Create the preset:
 
 ```shell
 kubectl apply -f https://k8s.io/examples/podpreset/preset.yaml
 ```
 
-Examine the created PodPreset:
+Verify that PodPreset has been created:
 
 ```shell
 kubectl get podpreset
 ```
 ```
-NAME             AGE
-allow-database   1m
+NAME             CREATED AT
+allow-database   2020-01-24T08:54:29Z
 ```
 
-The new PodPreset will act upon any pod that has label `role: frontend`.
+Here is the configuration file containing the definition of an applicable Pod:
```

Comment:
```suggestion
This manifest defines a Pod labelled `role: frontend` (matching the PodPreset's selector):
```

Reply:
That makes the intend of the example more obvious. Thanks for your suggestion